babel.config.js

```diff
@@ -0,0 +1,73 @@
+module.exports = function (api) {
```

Comment:
This comes from webpacker's upgrade guide:
https://github.com/rails/webpacker/blob/master/docs/v4-upgrade.md